app_dart/lib/src/service/scheduler.dart

```diff
@@ -462,6 +477,14 @@ class Scheduler {
     return getTargetsToRun(presubmitTargets, files);
   }
 
+  /// Returns `true` if [ciYaml.postsubmitTargets] should be ran during presubmit.
+  static bool _includePostsubmitAsPresubmit(CiYaml ciYaml, PullRequest pullRequest) {
```

Comment:
if (ciYaml.slug != Config.engineSlug) {
    return false;
}

For now until the pool usage can be evaluated and we can see the effect, if any on our SLO.